docs/reference/api/storage/HeapBlockDevice.md

```diff
@@ -1,2 +1,31 @@
 ## HeapBlockDevice
 
+The HeapBlockDevice class provides a way to simulate block devices for software development or testing. The created blocks are nonvolatile; they do not persist across power cycles.
+
+HeapBlockDevices have the following configurable parameters in either one of two constructors:
```

Comment:
Query: What are the two constructors?

Reply:
The second block that's talking about creating a block device using a block parameter to set the erase, program, and read sizes is the second constructor. The four parameters above that refer to the first constructor.

Reply:
@sg- Added subheaders to break out constructors, are the titles good or should that be rephrased?